plugins/friend-finder

```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/sololegends/runelite-friend-finder.git
+commit=d7f9ccbd28fe7724476e34a5a80d58e2242dac6f
```

Comment:
This plugin needs a `warning` for its integration with a third-party service, for example:
```suggestion
commit=d7f9ccbd28fe7724476e34a5a80d58e2242dac6f
warning=This plugin submits your rsn, player location and your IP address to a server not controlled or verified by the RuneLite developers.
```